baselines/dom.generated.d.ts

```diff
@@ -207,6 +207,7 @@ interface IDBObjectStoreParameters {
 }
 
 interface IntersectionObserverEntryInit {
+    isIntersecting: boolean;
     boundingClientRect?: DOMRectInit;
```

Comment:
These keys are incorrectly marked as optional; they're `required` in the spec.

It turns out that the generator script doesn't support `required` so I fixed this in https://github.com/Microsoft/TSJS-lib-generator/pull/271